src/runtime/components/CookieControl.vue

```diff
@@ -77,70 +77,76 @@
                       v-for="cookie in moduleOptions.cookies[cookieType]"
                       :key="cookie.id"
                     >
-                      <div class="cookieControl__ModalInputWrapper">
-                        <input
-                          v-if="
-                            cookieType === CookieType.NECESSARY &&
-                            cookie.name !== 'functional'
-                          "
-                          :id="resolveTranslatable(cookie.name, props.locale)"
-                          type="checkbox"
-                          disabled
-                          checked
-                        />
-                        <input
-                          v-else
-                          :id="resolveTranslatable(cookie.name, props.locale)"
-                          type="checkbox"
-                          :checked="
-                            getCookieIds(localCookiesEnabled).includes(
-                              getCookieId(cookie),
-                            )
-                          "
-                          @change="toogleCookie(cookie)"
-                        />
-                        <button @click="toggleButton($event)">
-                          {{ getName(cookie.name) }}
-                        </button>
-                        <label
-                          class="cookieControl__ModalCookieName"
-                          :for="resolveTranslatable(cookie.name, props.locale)"
-                          tabindex="0"
-                          @keydown="toggleLabel($event)"
-                        >
-                          {{ getName(cookie.name) }}
-                          <span v-if="cookie.description">
-                            {{ getDescription(cookie.description) }}
-                          </span>
-                          <span
+                      <slot name="cookie" v-bind="{ config: cookie }">
```

Comment:
I suppose this makes the cookie available under a `config` variable inside the component which takes this slot's place? If I'm right on that, why not name it `cookie` as well?

Reply:
that's right, it's because there is already "config" in the readme (https://github.com/dargmuesli/nuxt-cookie-control#cookie). The whole problem is that there is **cookie** slot in the docs, but there is no implementation 😄

Reply:
Ok, then let's change it to `cookie` as well in the documentation, ok? 
Also, the example given in the README does not work with this PR. `config` (or `cookie` therefore) is of type `Cookie` which is an object on which some `in` iteration as given in the README does not really make sense, I think. A `Cookie` does have an id, but no `cookies` property. I'd recommend to try that example out by running `pnpm dev` in the `/playground` directory and by including the example in the `app.vue` inside the same directory.